### PR TITLE
Transpose of matrix in java

### PR DESCRIPTION
**Pull request by raksha009**
issue no #1628 

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Addition of new code
- 

#### Languages Used:
- Java
- 

#### Files Added:
- Matrix_transpose.java
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Matrix_Operations/Java/Matrix_transpose.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// Java Program to calculation transpose of a matrix (2D Array)
+import java.util.Scanner;
+
+public class Matrix_transpose {
+
+	// Find transpose of matrix
+	public static void transpose_matrices(int[][] matrix) {
+		
+		int index1;
+		int index2;
```
Comment (raghav-dalmia) [inline Matrix_Operations/Java/Matrix_transpose.java]: club line 9 and 10.

File: Matrix_Operations/Java/Matrix_transpose.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		int[][] matrix_ans = new int[matrix.length][matrix[0].length];
+		for ( index1 = 0; index1 < matrix.length ; index1 ++) {
+			for ( index2 = 0; index2 < matrix[0].length; index2++ ) {
+				matrix_ans[index2][index1] = matrix[index1][index2];
+			}
+		}
+		
+		// Print the calculated matrix
+		for ( index1 = 0; index1 < matrix_ans.length ; index1 ++) {
```
Comment (raghav-dalmia) [inline Matrix_Operations/Java/Matrix_transpose.java]: remove space before ;

File: Matrix_Operations/Java/Matrix_transpose.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+		System.out.println(" Enter number of rows ");
+		int rows = s.nextInt();
+		
+		System.out.println(" Enter number of columns ");
+		int columns = s.nextInt();
+		int[][] matrix = new int[rows][columns];
+		
+		
+		System.out.println(" Enter elements of matrix ");
+		for (int i = 0 ; i < rows; i++) {
```
Comment (raghav-dalmia) [inline Matrix_Operations/Java/Matrix_transpose.java]: remove space before ;

File: Matrix_Operations/Java/Matrix_transpose.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+			for (int j = 0 ; j< columns; j++) {
+				matrix[i][j] = s.nextInt();
+			}
+		}
+
```
Comment (raghav-dalmia) [inline Matrix_Operations/Java/Matrix_transpose.java]: remove this extra line

File: Matrix_Operations/Java/Matrix_transpose.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+		
+		System.out.println(" Enter elements of matrix ");
+		for (int i = 0 ; i < rows; i++) {
+			for (int j = 0 ; j< columns; j++) {
+				matrix[i][j] = s.nextInt();
+			}
+		}
+			
+		transpose_matrices(matrix);
+
```
Comment (raghav-dalmia) [inline Matrix_Operations/Java/Matrix_transpose.java]: this one too

**Comment by raksha009** [comment]
@raghav-dalmia, made the changes

File: Matrix_Operations/Java/Matrix_transpose.java [hunk 6]
```diff
@@ -0,0 +1,63 @@
+// Java Program to calculation transpose of a matrix (2D Array)
+import java.util.Scanner;
+
+public class Matrix_transpose {
+
+	// Find transpose of matrix
+	public static void transpose_matrices(int[][] matrix) {
+		
+		int index1, index2;
+		int[][] matrix_ans = new int[matrix.length][matrix[0].length];
```
Comment (singh-shreya6) [inline Matrix_Operations/Java/Matrix_transpose.java:10]: Please remove extra spaces. Only 4 space indentation required from start of a new block.

File: Matrix_Operations/Java/Matrix_transpose.java [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+		int rows = s.nextInt();
+		
+		System.out.println(" Enter number of columns ");
+		int columns = s.nextInt();
+		int[][] matrix = new int[rows][columns];
+		
+		
+		System.out.println(" Enter elements of matrix ");
+		for (int i = 0; i < rows; i++) {
+			for (int j = 0 ; j< columns; j++) {
```
Comment (singh-shreya6) [inline Matrix_Operations/Java/Matrix_transpose.java:39]: Space before <

**Comment by singh-shreya6** [comment]
Also PR template, keep it as Fixes#issueNumber

**Comment by raksha009** [comment]
I have made the changes @singh-shreya6. Plz review. Also, can't do fixes issue cause the issue has my other PRs too under it.

**Comment by singh-shreya6** [comment]
That is okay. You can still add it like that. It will not close the issue. I have approved your code. Please squash your commits so that it can be merged.

**Comment by raksha009** [comment]
done @singh-shreya6

**Pull request merged**